includes/multi-currency/MultiCurrency.php

```diff
@@ -1354,14 +1354,15 @@ public function get_all_customer_currencies() {
 			Database_Cache::CUSTOMER_CURRENCIES_KEY,
 			function() {
 				global $wpdb;
-
-				$currencies = $wpdb->get_col(
-					"SELECT
-						DISTINCT(meta_value)
-					FROM
-						{$wpdb->postmeta}
-					WHERE meta_key = '_order_currency'"
-				);
+				$query_currencies_posts  = "SELECT DISTINCT(meta_value) FROM {$wpdb->postmeta} WHERE meta_key = '_order_currency'";
```

Comment:
Can you move these 2 declarations inside their respective branch ("if")? I think it's best to define variable only where they're being used

Reply:
On second thought, the idea of moving the queries as separate strings came from another class where the queries were really long and hard to read. It is not warranted here, so I've moved the queries back as strings within the `get_col` call. Hope that makes sense.